server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -251,13 +251,8 @@ public Form<EditTiClientInfoForm> updateClientInfo(
   public TrustedIntermediarySearchResult getManagedAccounts(
```

Comment:
Please update the Java docs of the method to include new changes like- "" .... If both
   are empty or incomplete dates are passed - an unfiltered list of accounts is returned..."

Reply:
Done